web_external/js/views/body/SourcePanel.js

```diff
@@ -20,17 +20,13 @@ minerva.views.SourcePanel = minerva.View.extend({
     displayWmsLayersList: function (evt) {
         var el = $(evt.currentTarget);
         var wmsSource = this.sourceCollection.get(el.attr('cid'));
-        if (!this.wmsLayersListWidget) {
-            this.wmsLayersListWidget = new minerva.views.WmsLayersListWidget({
-                el: $('#g-dialog-container'),
-                source: wmsSource,
-                collection: this.datasetCollection,
-                parentView: this
-            });
-            this.wmsLayersListWidget.render();
-        } else {
-            this.wmsLayersListWidget.setCurrentSource(wmsSource);
-        }
+        this.wmsLayersListWidget = new minerva.views.WmsLayersListWidget({
```

Comment:
@mgrauer, if I add this snippet just before line 23 then the dupe issue goes away:
```
        if (this.wmsLayersListWidget) {
            this.wmsLayersListWidget.destroy();
        }
```
But maybe this isn't as efficient as reusing the existing widget.  What do you think?